packages/core/helper-plugin/src/hooks/useClipboard.ts

```diff
@@ -1,7 +1,7 @@
 import { useCallback } from 'react';
 
 export const useClipboard = () => {
-  const copy = useCallback(async (value) => {
+  const copy = useCallback(async (value: string | number) => {
     try {
       // only strings and numbers casted to strings can be copied to clipboard
       if (typeof value !== 'string' && typeof value !== 'number') {
```

Comment:
If this hook will need to process user input, then this explicit typecheck can remain for runtime protection. However, if it gets all its values at build time then I can remove this typechecking to simplify things a bit more.

Reply:
i'd leave it, as a public facing hook i think being over protective is better and will reduce unintended crashes.